### DISCUSSION TITLE
How to change a short name?

### DISCUSSION THREAD

**Discussion by jazer143** [postalserver/postal#2062] [Help with using Postal]
Short name is grayed out upon creation of mail server. Where in the backend can change this? Thank you 

![short name](https://user-images.githubusercontent.com/105583467/178091423-6b1a7fb4-c184-479f-93ad-959dc0c35fe5.PNG)

**Comment by willpower232** [comment]
You should be able to change it directly in the database but I'm not sure what issues that might cause.
